seqr/views/react_app.py

```diff
@@ -43,11 +44,15 @@ def render_app_html(request, additional_json=None, include_user=True, status=200
         ui_version = 'local'
         html = html.replace('</head>', '<script defer="defer" src="/app.js"></script></head>')
 
+    should_show_loading_delay = bool(ANVIL_LOADING_DELAY_EMAIL_DATE)
+    if should_show_loading_delay:
+        should_show_loading_delay = datetime.strptime(ANVIL_LOADING_DELAY_EMAIL_DATE, '%Y-%m-%d') < datetime.now()
```

Comment:
How about moving the time checking to the front end?

Reply:
I'm not in love with javascript's date processing and I'd rather leave this sort of business logic server side